src/test/java/fr/insalyon/citi/golo/compiler/CompileAndRunTest.java

```diff
@@ -1618,4 +1610,32 @@ public void banged() throws Throwable {
     assertThat(result, not((Object)42));
 
   }
+
+  @Test
+  public void test_named_parameters() throws Throwable {
```

Comment:
I would also add a test case for the cases where some arguments are named, and some are not.